### ISSUE TITLE
Creating Changes.hs in the NoPCM example

### ISSUE THREAD

**Issue by elwazana** [JacquesCarette/Drasil#705]
@smiths @JacquesCarette @szymczdm  Hello, I've begun trying to add the Unlikely Changes to the NoPCM example in Drasil (as per issue https://github.com/smiths/caseStudies/issues/100), but I ran into a problem when trying to create a Changes.hs file to store the Likely and Unlikely Changes.

The Likely Changes reference the assumptions but then the Assumptions reference the likely changes, this creates an import loop between Assumptions.hs and Changes.hs. I thought about possibly forgoing the Changes.hs file and have its content defined in Body.hs.

But the README file for the SWHS example says that a Changes.hs file should exist, but the README for the NoPCM example doesn't mention a Changes.hs file. But I believe it would become a consistency issue if some examples have a Changes.hs while others don't.

**Comment by szymczdm** [comment]
Labels will fix this. Then we can have things separated cleanly.

**Comment by JacquesCarette** [comment]
I am not sure who is best to check on the state of this -- @smiths or @halonazhao ? Or whoever is working on the NoPCM case study?

**Comment by smiths** [comment]
@JacquesCarette, I think the import loop can be solved in the same way we are planning on solving it for the other import loops.  If we require that "likely changes" reference "assumptions" and then back calculate what references assumptions, we shouldn't have a loop.  The assumptions do not actually depend on likely changes, but the likely changes depend on assumptions.  I think once we get the back references listed for assumptions for the theoretical models (etc.), then we will be able to tackle this.

**Comment by JacquesCarette** [comment]
I agree - but that wasn't my point. We need someone to own to issue, who will check that it is done. Once we fix the label problem, this issue will not magically get fixed, a little bit of work will still need to be done. Who should that be?

**Comment by smiths** [comment]
I will take ownership of the issue of identifying the differences between Drasil and manual and divide this issue into sub-issues.  I will likely assign those issues to @halonazhao, or one of the other team  members, if @halonazhao is too busy.  Once the infrastructure is properly in place, and the inconsistency problems explicitly identified, I'm optimistic that the actual Drasil coding will be fast (for someone familiar with the DSLs).
